types/puppeteer/index.d.ts

```diff
@@ -1141,42 +1276,60 @@ export interface WaitForSelectorOptionsHidden extends WaitForSelectorOptions {
   hidden: true;
 }
 
-export interface FrameBase extends Evalable {
-  /**
-   * The method queries frame for the selector.
-   * If there's no such element within the frame, the method will resolve to null.
-   */
-  $(selector: string): Promise<ElementHandle | null>;
-
-  /**
-   * The method runs document.querySelectorAll within the frame.
-   * If no elements match the selector, the return value resolve to [].
-   */
-  $$(selector: string): Promise<ElementHandle[]>;
-
-  /**
-   * The method evaluates the XPath expression.
-   * @param expression XPath expression to evaluate.
-   */
-  $x(expression: string): Promise<ElementHandle[]>;
+export interface NetworkManager extends EventEmitter {
```

Comment:
Not public API.

Reply:
removed, but NetworkManager.d.ts stay because it contains `Response`, `ConsoleMessageLocation`, `Request`...